g-sgov/g-sgov-aktuality/g-sgov-aktuality-glosář-MaNe.ttl

```diff
@@ -31,27 +60,28 @@ g-sgov-aktuality-pojem:oznamovatel-jako-právnická-osoba
 g-sgov-aktuality-pojem:oznamovatel-jako-fyzická-osoba
         a       <http://www.w3.org/2004/02/skos/core#Concept> ;
         <http://www.w3.org/2004/02/skos/core#broader>
-                <https://slovník.gov.cz/generický/lidé/pojem/člověk> , g-sgov-aktuality-pojem:oznamovatel , <https://slovník.gov.cz/základní/pojem/typ-objektu> , <https://slovník.gov.cz/základní/pojem/role> ;
+                <https://slovník.gov.cz/veřejný-sektor/pojem/fyzická-osoba> , g-sgov-aktuality-pojem:oznamovatel , <https://slovník.gov.cz/základní/pojem/typ-objektu> , <https://slovník.gov.cz/základní/pojem/role> ;
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 g-sgov-aktuality:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
                 "Oznamovatel jako fyzická osoba"@cs .
 
 g-sgov-aktuality-pojem:aktualita
         a       <http://www.w3.org/2004/02/skos/core#Concept> ;
+        <http://purl.org/dc/terms/description>
+                "Společenská událost, kterou pořadatel pořádá pro skupinu účastníků"@cs ;
         <http://www.w3.org/2004/02/skos/core#broader>
-                <https://slovník.gov.cz/generický/věci/pojem/věc> , <https://slovník.gov.cz/základní/pojem/typ-objektu> ;
+                <https://slovník.gov.cz/veřejný-sektor/pojem/objekt-práva> , <https://slovník.gov.cz/základní/pojem/typ-objektu> ;
         <http://www.w3.org/2004/02/skos/core#inScheme>
                 g-sgov-aktuality:glosář ;
         <http://www.w3.org/2004/02/skos/core#prefLabel>
-                "Aktualita"@cs ;
-        <http://www.w3.org/2004/02/skos/core#scopeNote>
-                "Společenská událost, kterou pořadatel pořádá pro skupinu účastníků"@cs .
+                "Aktualita"@cs .
 
 g-sgov-aktuality:glosář
-        a               <http://www.w3.org/2004/02/skos/core#ConceptScheme> , owl:Ontology , <http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat/pojem/glosář> ;
+        a               <https://slovník.gov.cz/veřejný-sektor/pojem/glosář> , <http://www.w3.org/2004/02/skos/core#ConceptScheme> , owl:Ontology ;
+        rdfs:label      "Generický slovník aktualit - glosář"@cs ;
         <http://purl.org/dc/terms/created>
-                "2020-07-28" ;
+                "22.07.2020" ;
```

Comment:
Špatný formát datumu, jak si to tam dostal?

Reply:
Vyreseno. 

Exportoval jsem to starsi verzi modelio pluginu (nevedel jsem ze vznikl novy repozitar pluginu na githube) a tak jsem si stahoval zmeny z puvodni git-repo.